src/js/webrequest.js

```diff
@@ -263,23 +263,28 @@ function onHeadersReceived(details) {
   let tab_host = getHostForTab(tab_id);
   let response_host = window.extractHostFromURL(url);
 
+  let newHeaders = details.responseHeaders;
+  if (badger.isFlocOverwriteEnabled() && (details.type === 'main_frame' || details.type === 'sub_frame')) {
+    newHeaders.push({name: 'permissions-policy', value: 'interest-cohort=()'})
+  }
+
+  if (!badger.isPrivacyBadgerEnabled(tab_host)) {
+    return {responseHeaders: newHeaders};
+  }
```

Comment:
This block should be restored to its previous location, following `badger.logThirdPartyOriginOnTab()`.